netbench/netbench-cli/etc/Dockerfile

```diff
@@ -64,7 +65,18 @@ RUN set -eux; \
     cp target/debug/netbench-cli /app/bin; \
   fi; \
   rm -rf target; \
-  echo "#!/usr/bin/env bash\naws s3 cp s3://\$S3_BUCKET/client.json ./client.json\naws s3 cp s3://\$S3_BUCKET/server.json ./server.json\neval /usr/bin/netbench-cli ./client.json ./server.json \$@ > report.json\naws s3 cp ./report.json s3://\$S3_BUCKET/" > /app/bin/start; 
+  echo "#!/usr/bin/env bash\n \
+  set -e\n \
+  aws s3 cp s3://\$S3_BUCKET/client.json /tmp/client.json\n \
+  aws s3 cp s3://\$S3_BUCKET/exportedlogs/\$EXPORT_TASK_ID/ /tmp/ --recursive\n \
+  export DIR=\$(ls -d /tmp/*/)\n \
+  printenv\n \
+  gzip -d \$(echo \$DIR)000000.gz\n \
+  sort -k1 \"\$(echo \$DIR)000000\" > \$(echo \$DIR)000000_sorted\n \
+  echo \"\$(<\$(echo \$DIR)000000_sorted)\" |  awk -F'{' '{ st = index(\$0,\"{\");print \"{\" substr(\$0,st+1)}' > /tmp/server.json\n \
```

Comment:
what is this doing? maybe add some comments

Reply:
It's unzipping, cleaning, and sorting the exported server logs by timestamp. I will leave comments making this clear.